lib/screens/settings/eventsub_page.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/layout.dart';
+
+class EventConfigScreen extends StatelessWidget {
```

Comment:
Call this `EventsScreen` and update the filename to match? `Config` isn't really necessary because it's under `settings`.